### PR TITLE
Graphview

### PR DESCRIPTION
**Pull request by weipe**
When Luminus is started, you can access the graph viewer on http://127.0.0.1:4444/graphview.html in your local web browser. Known bug: if you want to load many graphs in sequence, please refresh the webpage between each switch - otherwise the height of the graph canvas keeps growing.

### CODE REVIEW

File: src/clj/jobtech_taxonomy_api/routes/services.clj [hunk 1]
```diff
@@ -66,6 +66,18 @@
      :current-user user
      (response/ok {:user user}))
 
+   (GET "/graph.json" []
```
Comment (qeshi) [inline src/clj/jobtech_taxonomy_api/routes/services.clj]: Why is this under the default namespace and not the public one?

Comment (qeshi) [reply src/clj/jobtech_taxonomy_api/routes/services.clj]: I think that maybe we should move the GET /relation/graph to the private part of the API, and the GET /relation/types to the public part of the API.

File: resources/public/js/graphview.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+// Yikes! An unhealthy load of global variables.
```
Comment (qeshi) [inline resources/public/js/graphview.js]: Why are you using javascript and not clojurescript 😉

File: src/clj/jobtech_taxonomy_api/routes/services.clj (hunk 1 above)
Comment (weipe) [reply src/clj/jobtech_taxonomy_api/routes/services.clj]: Good point, and it has a very simple explanation: I wanted to get the D3 graph up an running before implementing authentication in the simple web page. It is indeed possible to specify authentication headers in D3-requests, but you still need a little login form etc, so I just put it in default for now - let's refactor in some authentication and move it to private (perhaps after the demo).

File: resources/public/js/graphview.js (hunk 2 above)
Comment (weipe) [reply resources/public/js/graphview.js]: Very good point! It also has a very simple and shameful reason: I was lazy and found that using the D3 examples gave quick results. Let's fix this :)

**Pull request merged**